open-craft/opencraft
# BB-2241: Bug fixes

This PR fixes a few bugs in the new console:

1. Header customization background was pointing to the wrong variable.
2. External domain registration wasn't working, internal subdomain generation moved to python.

**Testing instructions for 2:**
1. Checkout this branch.
2. Create a new instance with an external domain, it should succeed.
3. Log in Ocim adming and check that a subdomain slug was properly generated (in this format `<domain-slug>-<random-hash>`).

**Reviewer:**
